modules/oc-mirror-disk-to-mirror.adoc

```diff
@@ -26,6 +26,9 @@ You can use the oc-mirror plugin to mirror the contents of a generated image set
 $ oc mirror --from=./mirror_seq1_000000.tar \// <1>
   docker://registry.example:5000             <2>
 ----
++
+For multi-arch release, append `--keep-manifest-list` option to oc command.
+
```

Comment:
the oc mirror plugin does not support mirroring manifestlists. this should be removed